server/tests/test_route_tickets.py

```
@@ -0,0 +1,144 @@
+    """Create test Flask application"""
+    app = Flask(__name__)
+    app.config['TESTING'] = True
+    app.config['SQLALCHEMY_DATABASE_URI'] = 'sqlite:///:memory:'
+    app.config['JWT_SECRET_KEY'] = 'test-secret-key'
+    app.config['WTF_CSRF_ENABLED'] = False
+
+    # Initialize extensions
+    db.init_app(app)
+
+    from flask_jwt_extended import JWTManager
+    jwt = JWTManager(app)
```

Ruff (0.13.1)

Local variable `jwt` is assigned to but never used

Remove assignment to unused variable `jwt`

(F841)